### PR TITLE
Analytics backend initial setup

### PR DESCRIPTION
**Pull request by sserurich**
- What does this PR do?
initial setup for the analytics back end service

- What Jiira task is related to this PR?
N/A
- How do I test out this PR?
pull the branch locally and follow the guides in README.md.txt
- Which end points should be ready for testing?

partial url : /api/v1/locations
                :/health

### CODE REVIEW

File: src/analytics/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.env
+env/
+.vscode
+__pycache__/
```
Comment (Baalmart) [inline src/analytics/.gitignore]: LGTM

File: src/analytics/README.md.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Install dependencies using the requirements.txt
+	-pip install -r requirements.txt
+
+set environment variables
+ windows
+ - set FLASK_APP=app.py
+ - set FLASK_ENV=development
+
+Run the Flask App
+    - flask run
```
Comment (Baalmart) [inline src/analytics/README.md.txt]: -Rename the file to remove the txt extension
- add the option for setting up the env variables for linux/mac

**Comment by pronapro** [comment]
LGTM

File: src/analytics/.gitignore (hunk 1 above)
Comment (sserurich) [reply src/analytics/.gitignore]: great

File: src/analytics/README.md.txt (hunk 2 above)
Comment (Priscilla-AL) [reply src/analytics/README.md.txt]: looks good

Comment (sserurich) [reply src/analytics/README.md.txt]: resolved

**Pull request merged**